CentralComputing/I2CManager.cpp

```diff
@@ -173,13 +173,13 @@ void I2CManager::check_for_sensor_error(const std::shared_ptr<I2CData> & check_d
   auto temp_arr = check_data->temp; 
   for (int index = 0; index < NUM_TMP; index++) {
     if (temp_arr[index] > (int) error_general_1_over_temp) {
-      // Send error 1 here
+      Command::set_error_flag(Command::SET_I2C_ERROR,I2C_OVER_TEMP_ONE);
```

Comment:
From my reading of this, it looks like only one of these errors could possible cause a trigger. The lowest "Error_general_X_over_temp"  will cause the error to be triggered.

Either have just 1 error, or check  different indexes with the different error conditions

Reply:
My bad I thought that there were 3 ascending errors (like a REALLY HIGH temp would be worse than a slightly high one)